### PR TITLE
Protofolio form

### PR DESCRIPTION
**Pull request by eng-mohamed-eldeeb**
Create a form using HTML and CSS. The form includes a name, e-mail, and a message text area. Add an "h1" and a "p" above the form.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -252,7 +252,7 @@ <h3>Contact Me</h3>
               <input type="email" class="userInfo" name="userMail" placeholder="You'r email">
```
Comment (Rustamxon7) [inline index.html:252]: ### ** GOOD JOB SO FAR 😄👌**

- [ ] Inside input email please also required attribute to make your form complete 🍀

File: style.css [hunk 2]
```diff
@@ -232,9 +232,9 @@ button:focus {
 }
```
Comment (Rustamxon7) [inline style.css:233]: - [ ] Your design looks good but it needs some improvements in code 🥲. You can do it just using flexbox in from to layout the elements 🏅

**Review by Rustamxon7 — CHANGES_REQUESTED** [review]
Hi @eng-mohamed-eldeeb , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @eng-mohamed-eldeeb, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**